pages/vi/faq.md

```diff
@@ -32,7 +32,7 @@
 
 #### Q3: Where can I find more information about this internship?
 
-+ More information about the internship can be found at the Virtual Software Engineer Intern listing [here](https://www.indeed.com/cmp/Open-Learning-Exchange,-Inc./jobs).
++ More information about the internship can be found at the Virtual Systems Engineer Intern listing [here](https://treehouses.io/#!index.md).
```

Comment:
fixed it by publishing again some openings ;)
https://www.indeed.com/cmp/Open-Learning-Exchange,-Inc./jobs